packages/theme/modules/catalog/category/components/sidebar/CategorySidebar.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <SkeletonLoader
     :is-loading="isLoading"
-    :height="'50vw'"
+    :height="'500px'"
```

Comment:
```suggestion
    height="500px"
```
Super nitpick, but now that you've touched it :P

Reply:
Feel free to commit with 
![image](https://user-images.githubusercontent.com/5359825/157659572-32bf27db-a9bd-4f50-9139-bbedff35f8fa.png)

Reply:
Not really, I am not sure how it suppose to work?